lib/identity.js

```diff
@@ -51,14 +51,22 @@ bedrock.events.on('bedrock-mongodb.ready', function init(callback) {
  * @param callback(err, record) called once the operation completes.
  */
 api.insert = function(actor, identity, callback) {
+  var meta = {};
   async.auto({
     checkPermission: function(callback) {
       api.checkPermission(
         actor, PERMISSIONS.IDENTITY_INSERT, {resource: identity}, callback);
     },
-    insert: ['checkPermission', function(callback) {
+    emit: ['checkPermission', function(callback) {
+      bedrock.events.emit('bedrock-identity.insert', identity, meta, callback);
```

Comment:
We've found that we should probably just be sending a single object to event handlers because it's less magical/guessy. So send "{identity: identity, meta: meta}".